frontend/src/components/Icons.tsx

```diff
@@ -343,6 +343,33 @@ export const ContactIcon = ({
   </svg>
 );
 
+/** Info button that inherits the text color of its container */
```

Comment:
suggestion: we should update this comment to reflect the icon's purpose.

Reply:
Whoops, sharp eye! e7a7775d